### PR TITLE
Load Team Info into Relay

### PR DESCRIPTION
**Pull request by vaage**
There was a TODO in the relay server to load team information
so that the relay would be able to accept read and write
requests from servers.

This change adds the ability to read a file that can contain
the team id and team secrets that it will accept.

Closes: #9

### CODE REVIEW

File: run_relay.sh [hunk 1]
```diff
@@ -14,5 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+TEAMS_FILE="$(pwd)/teams"
+
 cd './bin'
-java codeu.chat.RelayMain "2008"
+java codeu.chat.RelayMain '2008' "$TEAMS_FILE"
```
Comment (crepricg) [inline run_relay.sh]: Per Google's bash style guide, prefer bracket-quoting variables: https://google.github.io/styleguide/shell.xml?showone=Variable_expansion#Variable_expansion

File: src/codeu/chat/RelayMain.java [hunk 2]
```diff
@@ -44,22 +49,26 @@ public static void main(String[] args) {
 
     try (final ConnectionSource source = ServerConnectionSource.forPort(myPort)) {
 
+      final Server relay = new Server(1024, 16);
```
Comment (crepricg) [inline src/codeu/chat/RelayMain.java]: either use constants or inline comments to improve readability by specifying what these values mean.

File: run_relay.sh (hunk 1 above)
Comment (vaage-google) [inline run_relay.sh]: As we have not done this in any of our shell scripts, is now the right time to change how we do this?

Comment (crepricg) [inline run_relay.sh]: consistency >> styleguide. We will tackle all variables at once in a later CL.

Comment (vaage) [inline run_relay.sh]: Sounds good to me.

**Pull request merged**